api/python/tests/integration/data/local_manifest.jsonl

```diff
@@ -1,4 +1,4 @@
-{"semver": "1.1.1","comment": "example schema","top_hash": {"alg": "v0", "value": "c65a7b673b106c8051bd04db0976e84565dd1b5fc398f25b51eed79d2ff8bbfd"},"quilt_schema_version": "0.0.1"}
+{"semver": "1.1.1","comment": "example schema","quilt_schema_version": "0.0.1"}
```

Comment:
consider bumping `quilt_schema_version` : we should have a README where we document changes so that it's easy to write migrations when we need to.